src/ConnectAuth.ts

```diff
@@ -11,8 +17,10 @@ import {
 } from './types';
 
 interface CommonVerifyArgs<E, Q> {
-  baseUrl: string;
+  asymmetricKeyProvider?: KeyProvider;
```

Comment:
I've made it optional so it won't break implementations that don't use signed installations. However, the more I look at it, I think it makes sense to require it anyway since signed installs are a required migration. Thoughts?